sunkit_image/coalignment/__init__.py

```diff
@@ -0,0 +1,6 @@
+from sunkit_image.coalignment.interface import coalignment
+
+# To extend the package with new coalignment methods, please follow the import style shown below.
+from sunkit_image.coalignment.match_template import match_template_coalign  # noqa: F401
```

Comment:
```suggestion
from sunkit_image.coalignment.match_template import match_template_coalign as _
```
Does this avoid the bot check?